src/vesting-tokens-polkaswap.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Vesting tokens in SORA via Polkaswap | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Explore the vesting feature to explore the opportunity for gradual distribution of token access, providing full allocation of crypto assets over a specified period via the simple user interface.
+  - - meta
+    - name: keywords
+      content: SORA ecosystem, vesting tokens, distribute in period, gradual distribution, Business Cases for Vesting
+---
+
+# Vesting in Cryptocurrency
+
+Vesting is a method used in cryptocurrency projects to gradually release tokens over a specific period rather than distributing them all at once. Vesting is commonly applied to tokens allocated to project teams, early investors, and contributors, with the goal of encouraging long-term commitment and reducing the risk of sudden large sell-offs that could destabilize the token’s value.
+
+For more details, see[Vesting tokens in SORA](/vesting-tokens.md)
```

Comment:
doesn't work, / - means root, there maybe should be without /

Reply:
Fixed it